db/seeds.rb

```diff
@@ -5,3 +5,10 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+User.delete_all
+Message.delete_all
+Message.create(body: "привет", sender_id: 3, recipient_id: 1)
+Message.create(body: "пока", sender_id: 1, recipient_id: 3)
+10.times do |n|
+  User.create(email: "#{n+1}@#{n+1}.ru", password: '12345678', password_confirmation: '12345678')
```

Comment:
Line is too long. [97/80]<br>Surrounding space missing for operator '+'.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.